definitions/json/RoleToAccessProfiles/RoleToAccessProfiles.json

```diff
@@ -25,14 +32,14 @@
     "CaseTypeID": "ET_EnglandWales",
     "RoleName": "idam:caseworker-employment-api",
     "ReadOnly": "N",
-    "AccessProfiles": "caseworker-employment-api"
+    "AccessProfiles": "TTL_profile,caseworker-employment-api"
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "ET_EnglandWales",
     "RoleName": "idam:citizen",
     "ReadOnly": "N",
-    "AccessProfiles": "citizen"
+    "AccessProfiles": "TTL_profile,citizen"
```

Comment:
Assume we need similar for the professional user role as well

Reply:
sorted